client/webserver/locales/en-us.go

```diff
@@ -364,8 +362,12 @@ var EnUS = map[string]string{
 	"Backup App Seed":                 "Backup App Seed",
 	"seed_backup_implore": "It's highly recommended that you back up your application seed now. " +
 		"The application seed is critical to recovering your fidelity bonds and native wallets.",
-	"Backup Now":             "Backup Now",
-	"Skip this step for now": "Skip this step for now",
-	"save_seed_instructions": "Save the following seed somewhere safe. Do not share this with anybody.",
-	"Done":                   "Done",
+	"Backup Now":                "Backup Now",
+	"Skip this step for now":    "Skip this step for now",
+	"save_seed_instructions":    "Save the following seed somewhere safe. Do not share this with anybody.",
+	"Done":                      "Done",
+	"Reset App Password":        "Reset App Password",
+	"Forgot App Password":       "Forgot App Password",
+	"Download Application Seed": "Download Application Seed",
+	"seed_restore_warning":      "Make sure to use the correct seed for this application. Using the wrong seed will result in loss of funds.",
```

Comment:
There is a better way than displaying this warning, I think, it'll also address this concern you have:

> There might be some security concerns as any "valid" 128-char hex string can be used as a seed and there's no way to verify the reset pass seed except that wallets created with it will be different.

Upon seed generation (and for existing seeds, we can do that upon login once, to preserve backward compatibility) we can hash seed and store that hash in plain text on disk.
We can remove this warning tooltip and instead just hash the seed user supplied and compare it against that "original hash" we have stored on disk. If they match - it means user entered correct seed (the one he was using DEX with in the past), if they don't match we should warn the user suggesting:
- either to check and re-enter his seed again
- or use a **different flow** to hard-reset his DEX app entirely (this use-case is also a separate valid one, say if I downloaded DEX to just try it out, but didn't back up the seed and forgot the password I used ... I have no way to reset it, unless I delete DB manually)

Reply:
> We can remove this warning tooltip and instead just hash the seed user supplied and compare it against that "original hash" we have stored on disk. If they match - it means user entered correct seed (the one he was using DEX with in the past), if they don't match we should warn the user suggesting:

In https://github.com/decred/dcrdex/pull/2477/commits/eb467a4bcb8572c91c6f63053feb17a55d9f9d74, I opted to stick to our existing encryption pattern which requires a password to create a "crypter" and storing the necessary details to DB.

> use a different flow to hard-reset his DEX app entirely (this use-case is also a separate valid one, say if I downloaded DEX to just try it out, but didn't back up the seed and forgot the password I used ... I have no way to reset it, unless I delete DB manually)

I think in this scenario they will have to do the 'advanced stuff'. We could have a guide in place for this.

Reply:
> In https://github.com/decred/dcrdex/commit/eb467a4bcb8572c91c6f63053feb17a55d9f9d74, I opted to stick to our existing encryption pattern which requires a password to create a "crypter" and storing the necessary details to DB.

Your approach is even better!

> I think in this scenario they will have to do the 'advanced stuff'. We could have a guide in place for this.

In general, users don't read tutorials/text, not anymore (when they have a lot of apps that do stuff **for them**), so I'd treat a tutorial as a measure of last resort and I frankly think things like https://github.com/decred/dcrdex/pull/2468 is mostly a waste of time (might be useful for a couple of nerds, but that's it).
That said, this scenario of DEX re-creation is probably a rare one - hence doesn't deserve much attention any time soon (perhaps could be a separate issue lying around for someone, perhaps new contributor to the repo, to pick up down the line).